manifests/params.pp

```diff
@@ -22,7 +22,10 @@
       $virtinst_package = 'virtinst'
       $radvd_package = 'radvd'
       $sysconfig = false
-      $deb_default = {}
+      $deb_default = $::service_provider ? {
+        'systemd' => { 'libvirtd_opts' => '' },  # no '-d', it confuses systemd
```

Comment:
After applying this PR I get the following error:
```Error: Could not retrieve catalog from remote server: Error 400 on SERVER: Syntax error at '{'; expected '}' at /etc/puppet/environments/development/modules/libvirt/manifests/params.pp:26```

Some info about my environment:

```
h1 ~ # uname -a
Linux h1 4.4.0-36-generic #55-Ubuntu SMP Thu Aug 11 18:01:55 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

h1 ~ # lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:   	Ubuntu 16.04.1 LTS
Release:       	16.04
Codename:      	xenial

h1 ~ # puppet --version
3.8.5

h1 ~ # facter --version
2.4.6

h1 ~ # facter -p service_provider debian
debian => nil
service_provider => nil
```